autoPyTorch/evaluation/tae.py

```diff
@@ -110,90 +255,66 @@ def __init__(
         stats: Optional[Stats] = None,
         run_obj: str = 'quality',
         par_factor: int = 1,
-        output_y_hat_optimization: bool = True,
+        save_y_opt: bool = True,
```

Comment:
actually, I think `save_y_ensemble_optimization` or even `output_y_ensemble_optimization` as this flag also acts as a way to disable ensemble construction. As if it is `False` we cant build an ensemble. We can also add this to the description of the variable. Actually, it has got nothing to do with saving `y_opt`, these `true_targets_ensemble` are only used for ensemble construction